src/weekly/weekly.service.ts

```diff
@@ -412,16 +412,10 @@ export class WeeklyService {
       throw new HttpException('Not approver of user', HttpStatus.BAD_REQUEST);
     }
 
-    const week = await getConnection().getRepository(Week).findOne({
-      where: {
-        id: weekId
-      }
-    });
-
     const cred = await this.getGoogleCredentials();
     const oAuth2Client = await auth.authorize(cred);
 
-    const message = 'Timesheets: https://timesheets.opennetworking.org/\n\nPlease complete timesheet week: ' + week.begin + ' - ' + week.end;
+    const message = comment;
```

Comment:
I was thinking you might have the comment **in addition** to the link and the week start and end. 
But maybe you're putting these in to the comment already in the GUI. I'll leave it to you. Approved